client/js/models/archive.js

```diff
@@ -130,11 +130,7 @@ module.exports = {
       send('archive:initImportQueue', {archive}, noop)
     },
     import: function (data, state, send, done) {
-      const location = '/' + data
-      send('location:setLocation', { location }, done)
-      window.history.pushState({}, null, location)
-      send('archive:update', {entries: {}, numPeers: 0, downloadTotal: 0, uploadTotal: 0, size: 0}, noop)
-      send('archive:load', data, done)
```

Comment:
we actually need to change the location and trigger a server-render, which this was not doing before (it was just changing the url)